packages/core/package.json

```diff
@@ -24,8 +24,9 @@
       "types": "./dist/index.d.ts",
       "svelte": "./dist/index.js"
     },
-    "./theme": "./theme.ts",
-    "./prime.css": "./prime.css"
+    "./prime.css": "./prime.css",
+    "./plugins": "./plugins.ts",
+    "./theme": "./theme.ts"
```

Comment:
I didn't want to make any major changes as a part of this PR, but I am wondering if we want to combine these exports into a `tailwind.ts` file or something.

Reply:
There are some places where we import the theme directly to grab a color in JS, for example. It feels natural to grab colors from a file called "theme", but I don't feel terribly strongly. I appreciate keeping this PR more straightforward.

Reply:
I don't feel super strongly either, but in order to use PRIME you have to import both separately to include them in your config, which kind of feels weird as well. But you should only have to do that once, and you may be importing colors in multiple places, so keeping them separate is fine too. We could also make a `tailwind.ts` file that exports both `plugins` and `themes` and preserve the separate exports as well.